chicagorequests/__init__.py

```diff
@@ -128,12 +128,17 @@ def day_intervals(
 @click.option(
     "--list-request-types", is_flag=True, default=False, help="list valid request types"
 )
+@click.option(
+    "--save-json", is_flag=True, default=False, help="save output as json (datestamp as filename)"
```

Comment:
would it be better to let the user set a name?

Reply:
I thought about that. But then 1) you have to validate the name, 2) verify it's not overwriting another file, and 3) get into paths. Since the original was quick & dirty, I figured the file name would be the same. Avoids all the code overhead. They can rename after (I mean, you were asking them to use jq in the first place, so what's the difference), but this avoids collisions with a simple solution.